modules/repository/main.tf

```diff
@@ -0,0 +1,69 @@
+resource "github_repository" "this" {
+  name         = var.name
+  description  = local.description
+  homepage_url = local.homepage_url
+
+  visibility             = "public"
+  has_issues             = true
+  has_wiki               = false
+  has_projects           = false
+  allow_merge_commit     = false
+  allow_squash_merge     = true
+  allow_rebase_merge     = false
+  delete_branch_on_merge = true
+  has_downloads          = false
+  archived               = false
+  topics                 = local.topics
+  auto_init              = true
+  license_template       = "apache-2.0"
+
+}
+
+resource "github_branch" "default" {
+  repository = github_repository.this.name
+  branch     = "main"
```

Comment:
This needs to be configurable unless we're forcing them all over now?

Reply:
forcing them over :D

Reply:
Definitely worth a post in the channel about this before we post then.